packages/utils/src/crypto.ts

```diff
@@ -60,6 +61,20 @@ function keccak256Hash(data: string): string {
   return EthCrypto.hash.keccak256(data);
 }
 
+/**
+ * Hashes with the keccak256 algorithm with a normalization before and formats it
+ *
+ * @notice It will sort the object by keys before hashing
+ *
+ * @param data The data to hash
+ * @returns The hashed data multi-formatted
+ */
+function last20bytesOfNormalizeKeccak256Hash(data: any): string {
```

Comment:
```suggestion
function last20bytesOfNormalizedKeccak256Hash(data: any): string {
```

Reply:
or even, `getCurrencyHash`? isn't it what this does? in which case, `data` can be typed `ICurrency`